doc-pages/src/components/examples.js

```diff
@@ -0,0 +1,14 @@
+import React, {Component} from 'react';
+import ExampleApp from '../../../examples/main/app';
+
+// TODO(abm): This is made into a wrapper to improve later
+
+export default class Example extends Component {
```

Comment:
Yeah I wanted to redo the examples at some point to make it more seemless (e.g. same doc style whether you're running it through the docs or locally).